.github/actions/setup-dependencies/action.yml

```diff
@@ -86,7 +86,8 @@ runs:
 
     - name: Set MATLAB_HOME
       run: |
-        echo "MATLAB_HOME=/usr/local/MATLAB/R2023b/bin" >> $GITHUB_ENV
+        # echo "MATLAB_HOME=/usr/local/MATLAB/R2023b" >> $GITHUB_ENV
```

Comment:
Should we remove this?

Reply:
Actually, I wanted to check and see why this doesn't work. I tried `/usr/local/MATLAB/R2023b/` and `/usr/local/MATLAB/R2023b/bin`. It can't find the headers.

Reply:
I removed it.